scripts/tokenomics-emulation/network.ts

```diff
@@ -247,7 +247,7 @@ class Network {
     } else if (status.isFinalized) {
       const hash = status.asFinalized.toHex();
       console.info(`Transaction has been included in blockHash ${hash}`);
-      if (handleEvents !== undefined) {
+      if (typeof handleEvents !== 'undefined') {
```

Comment:
Why do we need this?

Reply:
To check the type of variable.

Reply:
What is wrong with the previous approach?

Reply:
reverted to previous state